tools/ITER_Profile_reader.py

```diff
@@ -0,0 +1,66 @@
+# This class provides routines for reading input profiles from CORSICA data files
+
+import numpy as np
+
+class ITER_Profile_reader:
+
+    def __init__(self, filename):
+        self.filename = filename
+        
+        f = open(self.filename,'r')
+        lines = f.readlines()
+        iLine = 0
+        keep_reading = True
+        
+        # Go through the file and find basic scalar data 
+        # (number of columns ncol for the profile datasets, 
+        # number of radial gridpoints N_Rho and the plasma current Ip)
+        while keep_reading:
+            line = lines[iLine]
+            #print(line)
+            if line.split():
+                if line.split()[0] == 'ncol':
+                    self.ncol = int(lines[iLine+1])
+                    iLine+=1
+                if line.split()[0] == 'N_Rho':
+                    self.N_Rho = int(lines[iLine+1])
+                    iLine+=1
+                    
+                if line.split()[0] == 'plasma' and line.split()[1] == 'current':
+                    self.Ip = float(lines[iLine+1].replace('D','e'))
+                    iLine+=1
+                            
+            
+            iLine+=1        
+            keep_reading = iLine<len(lines)
```

Comment:
It seems this is the only line modifying ``keep_reading``? If so, you could consider putting this boolean expression directly in the loop head instead.